docs/pages/core-concepts.mdx

```diff
@@ -216,6 +236,41 @@ users, roles, and resources, but the trust relationship allows users with certai
 in the root cluster to be mapped to roles and permissions defined in the leaf cluster.
 
 For more information about how to configure a trust relationship between clusters,
-see [Configure Trusted Clusters](admin-guides/management/admin/trustedclusters.mdx).
-For an overview of the architecture used in a trusted cluster relationship, see
-[Trusted Cluster Architecture](reference/architecture/trustedclusters.mdx).
+see [Configure Trusted Clusters](admin-guides/management/admin/trustedclusters.mdx). For an overview of the architecture used in a trusted cluster relationship, see [Trusted Cluster Architecture](reference/architecture/trustedclusters.mdx).
+
+## Teleport clients
+
+A **Teleport client** connects to a Teleport cluster to authenticate, request access, or interact with resources such as servers, databases, Kubernetes clusters, desktops, or internal web apps. Clients use short-lived certificates and Teleport’s Role-Based Access Control (RBAC) to ensure secure, auditable access. All clients rely on the Teleport Auth Service to verify identity and issue credentials, and they may connect through the Proxy Service when accessing resources behind firewalls or in private networks.
+
+### `tsh`
+
+`tsh` is the command-line tool used by Teleport users to authenticate, connect to infrastructure resources, and manage active sessions. 
+It supports SSH, Kubernetes, databases, and applications. This tool is commonly used by engineers and operators who prefer terminal-based workflows.
+
+### `tctl`
+
+`tctl` is the administrative command-line interface for managing a Teleport cluster. It allows administrators to configure roles, manage users, and work with resources such as tokens, trusted clusters, and audit logs. 
+It's typically used during initial cluster setup and for ongoing administrative tasks.
+
+### `tbot`
```

Comment:
Would it make sense to move the tbot discussion to where we talk about Machine ID?